Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -88,19 +93,19 @@ private static void TraktScrobble(PlexEvent evt, ScrobblePlayingStatus type)
         private void Scrobble(PlexEvent data)
         {
             PlexEvent.PlexMetadata metadata = data.Metadata;
-            (SVR_AnimeEpisode episode, SVR_AnimeSeries anime) = GetEpisode(metadata);
+            (SVR_AnimeEpisode episode, SVR_AnimeSeries anime) = GetEpisodeAndSeries(metadata);
```

Comment:
Why the name change?

Reply:
The name change was not needed, but I felt like the original name didn't really convey everything it did. It's reverted now.